### PR TITLE
 Add first 7 lessons of Basics section in Slovak language.

### PR DESCRIPTION
**Pull request by michalvalasek**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ author:
 title: Elixir School
 description: Lessons about the Elixir programming language
 baseurl: /
-url: https://elixirschool.com
+url: http://localhost:4000
```
Comment (doomspork) [inline _config.yml]: Hi @michalvalasek, could you remove this particular change?

**Comment by doomspork** [comment]
Thank you @michalvalasek, this is _awesome_.  We really appreciate all the work you've put into this.

File: _config.yml (hunk 1 above)
Comment (michalvalasek) [inline _config.yml]: Sure! Sorry for that, I completely forgot about it :)

**Comment by michalvalasek** [comment]
Glad to help, doing what I can to spread the Elixir love :)

**Comment by michalvalasek** [comment]
Glad to help, doing what I can to spread the Elixir love :)Glad to help, doing what I can to spread the Elixir love :)

**Comment by doomspork** [comment]
Everything looks good.  Thanks again @michalvalasek!  Looking forward to more contributions from you.

:tada:

**Pull request merged**